tailwind.config.js

```diff
@@ -41,6 +44,28 @@ module.exports = {
           logo: "#31CE9F",
         },
       },
+      fontSize: {
```

Comment:
These changes shouldn't be made. They're strongly not welcome. @porridgewithraisins @cool-pants